x/emissions/abci.go

```diff
@@ -12,28 +13,47 @@ import (
 
 func BeginBlocker(ctx sdk.Context, keeper keeper.Keeper) {
 
-	reservesFactor, bondFactor, durationFactor := keeper.GetBlockRewardComponents(ctx)
-	blockRewards := reservesFactor.Mul(bondFactor).Mul(durationFactor)
+	emissonPoolBalance := keeper.GetReservesFactor(ctx)
+	if emissonPoolBalance.IsZero() {
+		return
+	}
+	blockRewards, err := keeper.GetFixedBlockRewards()
+	if err != nil {
+		ctx.Logger().Error(fmt.Sprintf("Error while getting fixed block rewards %s", err))
+		return
+	}
 	if blockRewards.IsZero() {
+		ctx.Logger().Error("Block rewards are zero")
```

Comment:
Is it an error? I would change it to warn

Reply:
It should be an error , as the rewards are fixed , calculated using constants only